dask/dataframe/io/parquet.py

```diff
@@ -121,7 +117,8 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
                         columns=[c for c in pf.columns if c in dtypes])
 
     for cat in categories:
```

Comment:
No extra columns given to categories are just ignored.
The encoding of any given column can vary vary by *page*, which is a sub-unit of a row-groups, so a scan of all the data would be required to say for sure that everything is dictionary encoded. For files written by fastparquet, there is only ever one page per row group, but still.